go.mod

```diff
@@ -2,4 +2,9 @@ module github.com/beyondstorage/go-service-ftp
 
 go 1.15
 
-require github.com/beyondstorage/go-storage/v4 v4.4.0
+require (
+	github.com/beyondstorage/go-integration-test/v4 v4.2.0
+	github.com/beyondstorage/go-storage/v4 v4.4.0
+	github.com/jlaffaye/ftp v0.0.0-20210307004419-5d4190119067
```

Comment:
Let's better to depend on release version instead of git version, tracked at https://github.com/jlaffaye/ftp/issues/175